emails/tests/views_tests.py

```diff
@@ -28,10 +28,14 @@
     real_abs_cwd, 'fixtures', 'single_recipient_sns_body.json'
 )
 
+EMAIL_FIXTURES = [
+    'spamVerdict_FAIL',
+    'single_recipient',
+    'domain_recipient',
+    'single_recipient_list'
+]
 EMAIL_SNS_BODIES = {}
-for email_type in [
-    'spamVerdict_FAIL', 'single_recipient', 'domain_recipient'
-]:
+for email_type in EMAIL_FIXTURES:
     email_file = os.path.join(
         real_abs_cwd, 'fixtures', '%s_email_sns_body.json' % email_type
     )
```

Comment:
Next time we add a fixture, we should load all by filename:

```python
suffix = '_email_sns_body.json'
for email_file in glob.glob(os.path.join(real_abs_cwd, 'fixtures', '*' + suffix)):
    filename = os.path.basename(email_file)
    email_type = basename[:-len(suffix)]
    with open(email_file, 'r') as f:
        email_sns_body = json.load(f)
        EMAIL_SNS_BODIES[email_type] = email_sns_body
```

Reply:
did it this time!